main.css

```diff
@@ -97,3 +99,125 @@ nav ul .email-icon {
     display: block;
   }
 }
+
+/* Styling for headline  */
+
+.headline {
+  display: flex;
+  width: 100vw;
+  height: 870px;
+  align-items: center;
+  background-color: #1c1a19;
```

Comment:
Kindly make the `height: 870px` smaller, because there is a lot of space between the nav and the header

Your design
![image](https://user-images.githubusercontent.com/35535130/153339089-e67c41c3-10b8-4bf9-856c-d2f4c54065bb.png)
<br><br> 
Figma design
![image](https://user-images.githubusercontent.com/35535130/153339171-0379fc92-3e21-40c2-a9ea-8b7d4d954b71.png)